### DISCUSSION TITLE
download locations

### DISCUSSION THREAD

**Discussion by cirrusflyer** [tobychui/arozos#34] [General]
How about making the image file available from other locations?  It's taking over 8 hours to download to US.

**Comment by yeungalan** [comment]
Hi, can you try this one? I guess that should faster than ru one
https://github.com/tobychui/arozos/releases

**Reply by tobychui** [reply]
I think what he means is the Raspberry Pi image instead of the release file. However, without sponsor, it is quite expensive to setup a CDN in US or anywhere not Asia. @yeungalan you got any idea for setting up a CDN for the image?

**Reply by yeungalan** [reply]
AWS or Google Cloud Storage maybe is the good choice?

**Reply by PaperSama2333** [reply]
> Hi, can you try this one? I guess that should faster than ru one
> https://github.com/tobychui/arozos/releases

Thanks，ru is slowest in china，Also，chinamobie can't visit this web size:(

**Reply by yeungalan** [reply]
Since we don't have any ICP licenses, therefore I afraid that we can't provide any download mirror server located in China. 
Sorry for the inconvenience.

**Comment by tobychui** [comment]
It seems I have forgotten to update this discussion. 
I have added MediaFire for mirror services. You can always help seeds the image file (if you are into bt) or re-upload to other sites if it helps (Let me know and I will add your link to README)

Hope it helps :)
